kuma/wiki/forms.py

```diff
@@ -442,7 +443,20 @@ def akismet_error(self):
             document = self.instance.document
         except ObjectDoesNotExist:
             document = None
-        # wrapping this in a try/finally to make sure that even if
+
+        if exception and isinstance(exception, AkismetError):
+            # For Akismet errors, save the submission and exception details
+            dsa_params = parameters.copy()
+            dsa_params['_akismet_status_code'] = exception.status_code
```

Comment:
No, I guess not. I was trying to distinguish between the data that was sent and what was returned, but ``akismet_`` is as good of a prefix as ``_akismet_`` for that.  Will change.